src/Compilers/CSharp/Test/Semantic/Semantics/NullableReferenceTypesTests.cs

```diff
@@ -95651,6 +95648,241 @@ class Box<T>
                 Diagnostic(ErrorCode.ERR_BadBinaryOps, "x == null").WithArguments("==", "T", "<null>").WithLocation(53, 13));
         }
 
+        [Fact, WorkItem(34942, "https://github.com/dotnet/roslyn/issues/34942")]
+        public void ConditionalAccess_01()
+        {
+            var source =
+@"#nullable enable
+using System;
+
+public class Program
+{
+    static void Main()
+    {
+        string? value = ""42"";
```

Comment:
>""42""; [](start = 24, length = 7)

should start with a maybe-null null-state.

Reply:
Done

Reply:
`value` should start in maybe-null state. Otherwise the test doesn't have an effect (no-op).

---
In reply to: [290009777](https://github.com/dotnet/roslyn/pull/35953#discussion_r290009777) [](ancestors = 290009777)

Reply:
you may have missed this comment. At least this place (maybe others) where initial null-state makes the test less useful.

---
In reply to: [290069589](https://github.com/dotnet/roslyn/pull/35953#discussion_r290069589) [](ancestors = 290069589,290009777)

Reply:
ping

---
In reply to: [290078994](https://github.com/dotnet/roslyn/pull/35953#discussion_r290078994) [](ancestors = 290078994,290069589,290009777)

Reply:
This test is the repro given in #34942 but I will change it to be a parameter for clarity/simplicity